subjects/loafofbread/README.md

```diff
@@ -0,0 +1,44 @@
+# loafofbread
+
+### Instructions
+
+Write a function `LoafOfBread()` that takes a string and returns another one with words of 5 characters and skips the next character followed by newline `\n`.
+
+- If there is a space in the middle of a word it should ignore it and get the first character until getting to a length of 5.
+- If the string less than 5 characters returns "Invalid Output\n"
```

Comment:
I guess there is a missing `is`?

Reply:
It was indeed missing an "is". It is fixed.